src/redux/reducers/userReducer.ts

```diff
@@ -24,18 +27,24 @@ export const userReducer = (state: User = emptyUser, action: UserActions | Orcid
         .map(affiliationString => affiliationString.trim())
         .filter(affiliation => affiliation) as Affiliation[];
       const roles = action.user['custom:applicationRoles'].split(',') as RoleName[];
+      const isLoggedIn = true;
+      const email = action.user.email;
```

Comment:
personlig syns jeg ikke det trengs egne variabler for dette da de er såppas korte og bare brukes en plass. `affiliations` er såppas lang at jeg syns det gir mening der. men det er en smakssak, så whatever :)